doc/man/markdown/cockpit-desktop.1

```diff
@@ -0,0 +1,79 @@
+.\" Automatically generated by Pandoc 3.1.3
+.\"
+.\" Define V font for inline verbatim, using C font in formats
+.\" that render this, and otherwise B font.
+.ie "\f[CB]x\f[]"x" \{\
+. ftr V B
+. ftr VI BI
+. ftr VB B
+. ftr VBI BI
+.\}
+.el \{\
+. ftr V CR
+. ftr VI CI
+. ftr VB CB
+. ftr VBI CBI
+.\}
+.TH "cockpit-desktop" "1" "" "" "Cockpit Desktop integration"
+.hy
+.SH NAME
+.PP
+cockpit-desktop - Cockpit Desktop integration
+.SH SYNOPSIS
+.PP
+\ \ \ \f[B]cockpit-desktop\f[R] {URLPATH} [SSH_HOST]
+.SH DESCRIPTION
+.PP
+The \f[V]cockpit-desktop\f[R] program provides secure access to Cockpit
+pages in an already running desktop session.
+It starts a web server (\f[V]cockpit-ws\f[R]) and a web browser in an
+isolated network namespace, and a \f[V]cockpit-bridge(8)\f[R] in the
+running user session.
+.PP
+This avoids having to log into Cockpit, and having to enable
+\f[V]cockpit.socket\f[R] system-wide.
+The network isolation ensures that no other user, and not even other
+processes in the user\[aq]s session, can access this local web server.
+.PP
+\f[V]URLPATH\f[R] is the Cockpit page to open, i.
+e.
+the path component of Cockpit URLs.
+It is highly recommended to only open a particular page
+frame (https://cockpit-project.org/guide/latest/embedding.html), not the
```

Comment:
This is the render of markdown input produced by pandoc.

Url is written as a plain text, thus line breaks can happen in the middle of it.